.paket/Paket.Restore.targets

```diff
@@ -252,6 +253,10 @@
 				<AllowExplicitVersion>true</AllowExplicitVersion>
 
 			</PackageReference>
+
+			<PackageVersion Include="%(PaketReferencesFileLinesInfo.PackageName)">
+				<Version>%(PaketReferencesFileLinesInfo.PackageVersion)</Version>
+			</PackageVersion>
```

Comment:
can we mark this file as autogenerated?

Reply:
I tried now. Did it help?

![image](https://github.com/user-attachments/assets/bcffe12c-49b6-449d-9f44-253966275479)